modules/sandboxed-containers-viewing-workloads-from-cli.adoc

```diff
@@ -4,4 +4,30 @@
 
 [id="sandboxed-containers-viewing-workloads-from-cli_{context}"]
 
-= Viewing sandboxed containers workloads from the CLI
+= Viewing {sandboxed-containers-first} workloads from the CLI
+
+You can view the `runtimeClass` that the pods for your workloads use from the CLI.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+* Inspect the `runtimeClassName` field on the pod to see a pod running on {sandboxed-containers-first} versus a normal container.
++
+** On the node, each pod has a corresponding `qemu` process.
+
+.Verification
+
+* You can check the logs of the `openshift-sandboxed-containers-operator` controller pod to see detailed messages about the steps it is running.
```

Comment:
Do we need to tell users how to view the logs after getting the name of the pod?

Reply:
These were added as Verification steps temporarily as they will be moved into Troubleshooting in 4.9. For now, this is a good verification step.